python_modules/dagster/dagster/_daemon/run_coordinator/queued_run_coordinator_daemon.py

```diff
@@ -259,17 +263,49 @@ def _get_runs_to_dequeue(
             tag_concurrency_limits_counter = TagConcurrencyLimitsCounter(
                 tag_concurrency_limits, in_progress_runs
             )
-
             batch += queued_runs
             batch = self._priority_sort(batch)
 
+            if instance.run_coordinator.should_block_op_concurrency_limited_runs:
+                try:
+                    global_concurrency_limits_counter = GlobalOpConcurrencyLimitsCounter(
+                        instance,
+                        batch,
+                        in_progress_run_records,
+                        instance.run_coordinator.op_concurrency_slot_buffer,
+                    )
+                except:
+                    self._logger.error(
+                        "Failed to initialize op concurrency counter: \n%s",
+                        serializable_error_info_from_exc_info(sys.exc_info()),
+                    )
```

Comment:
self._logger.exception("Failed to initialize op concurrency counter) will include that context automatically

Reply:
👍